docs/03-intermediate/03-wait-wake-system/_category_.yml

```diff
@@ -0,0 +1,2 @@
+label: Asynchronous logic 
```

Comment:
wait_and_wake file: 
"These changes are stored in the program's persistent storage, so they are not lost when the program is paused"
Please rephrase it specifying that not program paused, but exact message actually, because we have a concept of paused programs

Reply:
and what does it mean to be paused

Reply:
and explain there more about rents and deadlines of being in WL

Reply:
and wakes doesnt return errors on "incorrect ids". it only returns errors in case of duplicate wakes within the same execution

Reply:
and somehow highlight important parts about state saving and further execution from the beginning